test_runner/src/main/kotlin/ftl/reports/CostReport.kt

```diff
@@ -28,21 +24,6 @@ object CostReport : IReport {
             totalBillablePhysicalMinutes += it.billableMinutes.physical
         }
 
-        val virtualCost = calculateVirtualCost(totalBillableVirtualMinutes.toBigDecimal())
```

Comment:
Removed because currently causing failed tests. We will merge all events into single https://github.com/Flank/flank/issues/2087 so this event is currently redundant